lib/tasks/xapi.rake

```diff
@@ -1,8 +1,9 @@
 namespace :xapi do
   desc "seed database with test data for x-api acceptance tests"
-  task :seed do
-    require 'bundler'
-    Bundler.require
+  task seed: [:connection] do
+    # User#save requires Exercism.uuid
+    # Submission#before_create requires Exercism.uuid
+    # UserExercise#before_create requires Exercism.uuid
```

Comment:
May I ask why?
I let the comment there to let it know why is used the `require 'exercism'` line. Hoping to remove the comments once we remove the dependency on `Exercism.uuid`.